Mathlib/Tactic/RestateAxiom.lean

```diff
@@ -0,0 +1,85 @@
+/-
+Copyright (c) 2017 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+Ported by: E.W.Ayers
+-/
+import Lean
+import Lean.Data
+
+open Lean
+open Lean.Elab
+open Lean.Elab.Command
+
+def restate_axiom_core (decl_name : Name) (new_name : Option Name := none)
```

Comment:
The new naming convention has `restate_axiom_core`, `restate_axiom_syntax` and `restate_axiom_cmd` in camel case.

Reply:
I should keep the keyword command as `restate_axiom` though right?

Reply:
Yes, like the `syntax` from the original file. "keyword-like" things like tactics and commands are snake-cased, definitions are camel cased.